### PR TITLE
Updating GLOW contacts, description, websites

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

File: virtual-organizations/GLOW.yaml [hunk 1]
```diff
@@ -1,27 +1,30 @@
-AppDescription: GLOW applications are diverse.  Minimal requirements are machines
-  with ~500MB RAM and working scratch space of ~500MB per job.  The GLOW VO is managed
+AppDescription: Applications from UW-Madison (GLOW) applications are diversem with minimal
```
Comment (matyasselmeci) [inline virtual-organizations/GLOW.yaml]:
```suggestion
AppDescription: Applications from UW-Madison (GLOW) applications are diverse with minimal 
```

File: virtual-organizations/GLOW.yaml [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+  - ID: 29a39113245cb237d6377e22152a1cb2585445e3
+    Name: Lauren Michael
   Registration Authority:
-  - ID: 210d07bf782db9938049eaf11820ec908fdda3b5
-    Name: Daniel Bradley
   - ID: 46a55ac4815b2b8c00ff283549f413113b45d628
     Name: Aaron Moate
-  Security Contact:
-  - ID: 210d07bf782db9938049eaf11820ec908fdda3b5
+   - ID: 210d07bf782db9938049eaf11820ec908fdda3b5
```
Comment (matyasselmeci) [inline virtual-organizations/GLOW.yaml]:
```suggestion
  - ID: 210d07bf782db9938049eaf11820ec908fdda3b5
```

File: virtual-organizations/GLOW.yaml (hunk 1 above)
Comment (matyasselmeci) [inline virtual-organizations/GLOW.yaml]:
```suggestion
AppDescription: Applications from UW-Madison (GLOW) applications are diverse with minimal 
```

File: virtual-organizations/GLOW.yaml [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
     Name: Daniel Bradley
+  Security Contact:
   - ID: 46a55ac4815b2b8c00ff283549f413113b45d628
     Name: Aaron Moate
+   - ID: 210d07bf782db9938049eaf11820ec908fdda3b5
```
Comment (matyasselmeci) [inline virtual-organizations/GLOW.yaml]:
```suggestion
  - ID: 210d07bf782db9938049eaf11820ec908fdda3b5
```

File: virtual-organizations/REPORTING_GROUPS.yaml [hunk 4]
```diff
@@ -397,12 +397,14 @@ glast.org:
...
   Contacts:
-  - ID: 48e1c2f26dc3479a6cf9b2de7c79d654ac27b1d1
-    Name: Miron Livny
+  - ID: 46a55ac4815b2b8c00ff283549f413113b45d628
+    Name: Aaron Moate
   - ID: 29a39113245cb237d6377e22152a1cb2585445e3
     Name: Lauren Michael
   - ID: 6227448ebe06b0e0e9ad3538508ad295d5933dc9
     Name: Greg Thain
+   - ID: 48e1c2f26dc3479a6cf9b2de7c79d654ac27b1d1
```
Comment (matyasselmeci) [inline virtual-organizations/REPORTING_GROUPS.yaml]:
```suggestion
  - ID: 48e1c2f26dc3479a6cf9b2de7c79d654ac27b1d1
```

File: virtual-organizations/GLOW.yaml [hunk 5]
```diff
@@ -40,10 +43,10 @@ OASIS:
       ID: 46a55ac4815b2b8c00ff283549f413113b45d628
   UseOASIS: true
 PrimaryURL: http://www.chtc.cs.wisc.edu
-PurposeURL: http://www.cs.wisc.edu/condor/glow/index.html
+PurposeURL: http://www.chtc.cs.wisc.edu/Approach.html
```
Comment (matyasselmeci) [inline virtual-organizations/GLOW.yaml]:
```suggestion
PurposeURL: http://www.chtc.cs.wisc.edu/approach.shtml
```

**Comment by lmichael107** [comment]
suggestions accepted

**Pull request merged**